packages/commonwealth/client/scripts/views/components/component_kit/CWSidebarHeader/CWSidebarHeader.tsx

```diff
@@ -27,7 +27,13 @@ const SidebarHeader = ({
         }
       />
 
-      <CWText className="header" type="h5">
+      <CWText
+        className="header"
+        type="h5"
+        onClick={() =>
```

Comment:
this works. But I suggest using the `<Link/>` element from the react router.

Reply:
link won't handle custom domain properly